README.md

```diff
@@ -27,15 +24,13 @@ sourceControl {
 
 // build.gradle:
 // use the name of a specific tag from https://github.com/dynatrace-oss/opentelemetry-metric-java/tags
-def dynatraceMetricsExporterVersion = "v0.5.0"
+def dynatraceMetricsExporterVersion = "v1.0.0"
 
 dependencies {
     implementation("com.dynatrace.opentelemetry.metric:dynatrace:${dynatraceMetricsExporterVersion}")
 }
 ```
 
-Gradle pulls the library in the specified version directly from GitHub and includes it.
```

Comment:
Just wondering, will the released version be published to maven central? :thinking:

Reply:
No, we discussed that internally and will continue with the source dependency. Seems to work fine!